### PR TITLE
Fix loot table for Sealed Scroll Case (i.33926)

### PR DESCRIPTION
**Pull request by f0n1x**
Source: https://web.archive.org/web/20090208155218/http://wow.allakhazam.com/db/item.html?witem=33926#Contains_Items
https://www.wowhead.com/wotlk/item=33926/sealed-scroll-case#contains

### CODE REVIEW

File: Updates/4818_sealed_scroll_case_i.33926.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+(33926, 27500, 0, 1, 1, 5, 0, 'Scroll of Protection V'),
+(33926, 27501, 0, 1, 1, 5, 0, 'Scroll of Spirit V'),
+(33926, 27502, 0, 1, 1, 5, 0, 'Scroll of Stamina V'),
+(33926, 27503, 0, 1, 1, 5, 0, 'Scroll of Strength V'),
+(33926, 33457, 0, 1, 1, 5, 0, 'Scroll of Agility VI'),
+(33926, 33458, 0, 1, 1, 5, 0, 'Scroll of Intellect VI'),
+(33926, 33459, 0, 1, 1, 5, 0, 'Scroll of Protection VI'),
+(33926, 33460, 0, 1, 1, 5, 0, 'Scroll of Spirit VI'),
+(33926, 33461, 0, 1, 1, 5, 0, 'Scroll of Stamina VI'),
+(33926, 33462, 0, 1, 1, 5, 0, 'Scroll of Strength VI');
```
Comment (AnonXS) [inline Updates/4818_sealed_scroll_case_i.33926.sql:20]: please change mincount and maxcount, 5-5 for V and 6-8 for VI.

grafik

File: Updates/4818_sealed_scroll_case_i.33926.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
+-- fix loot table for i.33926 (Sealed Scroll Case)
+-- https://web.archive.org/web/20090208155218/http://wow.allakhazam.com/db/item.html?witem=33926#Contains_Items
+
+-- remove refloot since we cannot group refloots
+DELETE FROM `item_loot_template` WHERE entry = 33926 AND item = 50604;
+
+-- add all scrolls V/VI individually to Sealed Scroll Case loot table
+INSERT INTO `item_loot_template` (entry, item, ChanceOrQuestChance, groupid, mincountOrRef, maxcount, condition_id, comments) VALUES
```
Comment (AnonXS) [inline Updates/4818_sealed_scroll_case_i.33926.sql:8]: Please add a delete query befor this insert, so it is easier to port/reexeceute.

**Comment by f0n1x** [comment]
Fixed!

**Comment by Grz3s** [comment]
reason why you closed this?

**Comment by f0n1x** [comment]
cleaning it up and resubmitted after convo with anon

**Pull request closed without merging**